observability/src/main/java/io/grpc/observability/interceptors/LogHelper.java

```diff
@@ -225,6 +236,12 @@ void logTrailer(
         .setEventLogger(eventLogger)
         .setLogLevel(LogLevel.LOG_LEVEL_DEBUG)
         .setRpcId(rpcId);
+    if ((pair != null) && (pair.size != 0)) {
```

Comment:
nit: the parens around `pair != null` etc are not required.

Reply:
Removed extra parentheses.